tesseract_urdf/include/tesseract_urdf/box.h

```diff
@@ -28,86 +28,19 @@
 
 #include <tesseract_common/macros.h>
 TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
-#include <tesseract_common/status_code.h>
-#include <tesseract_common/utils.h>
-#include <Eigen/Geometry>
 #include <tinyxml2.h>
-#include <boost/algorithm/string.hpp>
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 #include <tesseract_geometry/impl/box.h>
```

Comment:
I went back and forth on this but I think it would be good to do this. The urdf_parser is the only thing that should be used externally so it would be good to push the headers to the cpp. I will make the change.

Reply:
Done.